components/collection/drop/GenerativeLayout.vue

```diff
@@ -127,6 +130,9 @@ const props = withDefaults(
   },
 )
 
+const { width } = useWindowSize()
```

Comment:
why do we introduce this again? unrelated changes?

ref: https://github.com/kodadot/nft-gallery/issues/8767

Reply:
not unrelated at all

see my notes above in PR description